influxql/functions.go

```diff
@@ -415,16 +417,35 @@ func ReduceCountDistinct(values []interface{}) interface{} {
 	return len(index)
 }
 
+type NumberType int8
+
+const (
```

Comment:
At first glance this seems to overlap with the `InspectDataType` in the same package. Not exactly the same, but it seems redundant.